### PR TITLE
stop triggering multiple nightlies off the same revision

### PR DESCRIPTION
**Pull request by escapewindow**
### Pull Request checklist

- [ ] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/android-components/blob/main/docs/changelog.md) or does not need one
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/main/android/accessibility_guide.md) or does not include any user facing features

### After merge
- **Milestone**: Make sure issues closed by this pull request are added to the [milestone](https://github.com/mozilla-mobile/android-components/milestones) of the version currently in development.
- **Breaking Changes**: If this is a breaking change, please push a draft PR on [Reference Browser](https://github.com/mozilla-mobile/reference-browser) to address the breaking issues.

### CODE REVIEW

File: taskcluster/ac_taskgraph/target_tasks.py [hunk 1]
```diff
@@ -3,13 +3,38 @@
...
 @_target_task("nightly")
 def target_tasks_nightly(full_task_graph, parameters, graph_config):
     def filter(task, parameters):
         return task.attributes.get("build-type", "") == "nightly"
 
+    index_path = (
+        f"{graph_config['trust-domain']}.v2.{parameters['project']}.branch."
+        f"{parameters['head_ref']}.revision.{parameters['head_rev']}.taskgraph.decision-nightly"
+    )
+    if os.environ.get("MOZ_AUTOMATION") and retry(
```
Comment (jcristau) [inline taskcluster/ac_taskgraph/target_tasks.py:29]: `retry` isn't defined here AFAICT.

**Review by jcristau — CHANGES_REQUESTED** [review]
Looks like this is missing an import and requirements addition.

**Review by jcristau — APPROVED** [review]
Thanks!

File: taskcluster/test/params/main-repo-push-master.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+base_repository: https://github.com/mozilla-mobile/android-components
+base_rev: 77293b6bb9624749aa421e627f812613aa982557
+build_date: 1570437923
+do_not_optimize: []
+existing_tasks: {}
+filters:
+- target_tasks_method
+head_ref: refs/heads/master
```
[the remark this answers is not in the document]
Comment (escapewindow) [reply taskcluster/test/params/main-repo-push-master.yml:8]: Good catch! Will do.

**Pull request merged**